Centaurus.Domain/Quanta/Handlers/QuantumHandler.cs

```diff
@@ -176,6 +178,16 @@ async Task<ResultMessage> AuditorHandleQuantum(MessageEnvelope envelope)
                 return result;
         }
 
+        void ValidateAccountRequestRate(MessageEnvelope envelope)
+        {
+            var request = envelope.Message as RequestQuantum;
+            if (request == null)
+                return;
+            var account = Global.AccountStorage.GetAccount(request.RequestMessage.Account);
```

Comment:
This can be a minor performace problem in a future. We populate the account when the request arrives. Can we somehow obtain it from the request, without fetching from the storage?

Reply:
Let's move it to separate issue

Reply:
Ref https://github.com/centaurus-project/centaurus/issues/39